lib/mayaUsd/ufe/UsdTransform3dCommonAPI.cpp

```diff
@@ -278,9 +278,10 @@ Ufe::Transform3d::Ptr
 UsdTransform3dCommonAPIHandler::transform3d(const Ufe::SceneItem::Ptr& item) const
 {
     UsdSceneItem::Ptr usdItem = std::dynamic_pointer_cast<UsdSceneItem>(item);
-#if !defined(NDEBUG)
-    assert(usdItem);
-#endif
+
+    if (!usdItem) {
```

Comment:
All files have the same fix.  Strictly speaking, only the first handler in the Transform3d chain of responsibility needs the fix, but more robust to fix all Transform3d handlers.

Transform3d handlers for UFE v2 are different from the UFE v1 handler (there is no chain of responsibility in UFE v1).